### ISSUE TITLE
[Feature Request] automatically generate pulications in the CV from .bib file

### ISSUE THREAD

**Issue by mingwei-liu** [alshedivat/al-folio#2167]
### Have you checked that your feature request isn't already filed?

- [X] I read through [FAQ](https://github.com/alshedivat/al-folio/blob/master/FAQ.md) and searched through the [past issues](https://github.com/alshedivat/al-folio/issues), none of which addressed my feature request.
- [X] Yes, I have checked that this feature request isn't already filed.

### Description & Motivation

Thanks for the great work! When creating the cv page, I have to add the publications manually one by one. Is it possible to use the previous pubilications.bib file to automatically generate it, similar to the the publications pages?

**Comment by george-gca** [comment]
Yes it is. For this you should create another file like [_layouts/bib.liquid](https://github.com/alshedivat/al-folio/blob/master/_layouts/bib.liquid) that will be responsible for displaying the bib information in the correct format, and then use the bibliography tag in the cv page, overriding the default bibliography template by passing the `--template` or `-T` parameter. For more information see [jekyll-scholar](https://github.com/inukshuk/jekyll-scholar?tab=readme-ov-file#bibliography-template) documentation.

If you do this modification, would you mind sending a PR?

**Comment by mingwei-liu** [comment]
Sure. I will give it a try.

**Comment by gcostantino** [comment]
hello, I am trying to do that as well, but I don't manage to "print" the publications inside a "publicatiosn" tag inside the cv.yml. I had tried something like:

- title: Publications
  contents:
    bibliography_template: bib

but this does not work. It just prints "{"bibliography_template"=>"bib"}" inside my cv. Could you please help me? Thanks!

**Comment by george-gca** [comment]
To make this work you must implement it like I said before.

**Comment by HangJung97** [comment]
Any update on this? This feature would be useful.

**Comment by george-gca** [comment]
We are accepting PRs on this matter. If you manage to implement this, it will be a good contribution.
